### PR TITLE
Selenium upgrade to fix download issues

### PR DESCRIPTION
**Pull request by jmthibault79**
Copy of https://github.com/broadinstitute/firecloud-ui/pull/1089

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/project/Dependencies.scala [hunk 1]
```diff
@@ -33,7 +36,7 @@ object Dependencies {
     "com.typesafe.akka"   %%  "akka-slf4j"          % akkaV,
     "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
     "org.scalatest"       %%  "scalatest"     % "2.2.6"   % "test",
-    "org.seleniumhq.selenium" % "selenium-java" % "2.35.0" % "test",
+    "org.seleniumhq.selenium" % "selenium-java" % "3.8.1" % "test",
```
Comment (rtitle) [inline automation/project/Dependencies.scala]: nice

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/util/WebBrowserUtil.scala [hunk 2]
```diff
@@ -144,7 +144,7 @@ trait WebBrowserUtil extends WebBrowser {
 
     private def withWaitForCondition(timeOutInSeconds: Long)(f: => Boolean)(implicit webDriver: WebDriver): Boolean = {
       val wait = new WebDriverWait(webDriver, timeOutInSeconds)
-      wait until new ExpectedCondition[Boolean] {
+      wait until new java.util.function.Function[WebDriver, Boolean] {
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/util/WebBrowserUtil.scala]: Ugh, no scala DSL for this?

**Comment by rtitle** [comment]
Looks good, :shipit: if tests pass. Good to stay current on Selenium versions.

**Comment by helgridly** [comment]
👍

**Pull request merged**